rcl/src/rcl/logging_rosout.c

```diff
@@ -153,8 +154,10 @@ rcl_ret_t rcl_logging_rosout_init_publisher_for_node(
     return RCL_RET_ERROR;
   }
   if (rcutils_hash_map_key_exists(&__logger_map, &logger_name)) {
-    RCL_SET_ERROR_MSG("Logger already initialized for node.");
-    return RCL_RET_ALREADY_INIT;
+    // @todo: nburek enfornce node name unique
```

Comment:
Aside from the typo, which I was instinctively fixing when I stopped to think, is this TODO for enforcing node-name uniqueness categorically or following up on the node-name uniqueness design question (https://github.com/ros2/design/issues/187)

I think the TODO would be clearer if it included a link to that issue, if indeed that's what it is intended to regard.

Reply:
Beside the text of the todo the annotation used in the rest of code base is: `TODO(name) ...` instead of `@todo: name ....`